drafttopic/__init__.py

```diff
@@ -1,3 +1,17 @@
+"""
+This package contains a set of utilities and assets for predicting topics to
+new drafts based on Wikiprojects on English Wikipedia.
+
+In this package, you'll find the
+:mod:`feature lists <drafttopic.feature_lists>` used to train models
+for the supported wiki.  There's a set of
+:mod:`command-line utilities <drafttopic.utilities>`
+that are used to perform data pipeline operations specific to training and
+testing draft t opic models.
+
+See the :ref:`API Reference <api_reference>` for low level details.
```

Comment:
I don't think this `:ref:` actually links to anything.

Reply:
fixed it!